File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        StringBuilder sb = new StringBuilder();
        
        for (int i = 0; i < my_string.length(); i++) {
            char ch = my_string.charAt(i);
            
            if (ch >= 'A' && ch <= 'Z') {
                sb.append((char)(ch + 32));
            } else {
                sb.append((char)(ch - 32));
            }
        }
        return sb.toString();
    }
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.cpp
#include <string>
#include <vector>
#include <cctype>

using namespace std;

string solution(string my_string) {
    string answer = "";
    for(char c : my_string) {
        if (islower(c)) {
            answer += toupper(c);
        } else {
            answer += tolower(c);
        }
    }
    return answer;
}